PersonalWebsite/input/index.md

```diff
@@ -223,6 +236,7 @@ Note: This list is maintained mostly by me and random contributions. Data will i
         <td>✅Yes</td>
         <td>❌No</td>
         <td>❌No</td>
+        <td>🟠Only versions uploaded to CurseForge</td>

```

Comment:
```suggestion
        <td>❌No</td>

```
legacy FTB doesn't refer to specific packs, but a old API, that FTB is still operating.
see the legacy FTB section in PrismLauncher.